lib/LANraragi/Plugin/Metadata/Hdoujin.pm

```diff
@@ -1,107 +1,170 @@
-package LANraragi::Plugin::Metadata::Hdoujin;
+package LANraragi::Plugin::Metadata::HDoujin;
```

Comment:
Changing the namespace is fine but I believe you'll have to change the tests to match

Reply:
Done!